src/libraries/System.Runtime.InteropServices/src/System/Runtime/InteropServices/IDispatchImplAttribute.cs

```diff
@@ -4,7 +4,7 @@
 namespace System.Runtime.InteropServices
 {
     [AttributeUsage(AttributeTargets.Class | AttributeTargets.Assembly, Inherited = false)]
-    [Obsolete("This attribute is deprecated and will be removed in a future version.", error: false)]
+    [Obsolete("IDispatchImplAttribute has deprecated and is not supported.", error: false)]
```

Comment:
```suggestion
    [Obsolete("IDispatchImplAttribute has been deprecated and is not supported.", error: false)]
```

Reply:
This was the only occurrence of that mistake. Thanks for catching it!